evm/src/cpu/kernel/asm/account_code.asm

```diff
@@ -0,0 +1,130 @@
+%macro extcodehash
+    // stack: address
+    %mpt_read_state_trie
+    // stack: account_ptr
```

Comment:
Similar to the comment in #797, I think we need to handle `account_ptr == null` separately. I think we can return 0, that would match `EXTCODEHASH` semantics.

Reply:
(Unless you plan to check for account existence outside of this? If so let's comment that it assumes the account exists.)